### ISSUE TITLE
[1.16.4] Chat Delay

### ISSUE THREAD

**Issue by elkoyofficial** [kangarko/ChatControl#489]
Minecraft Version: 1.16.4
MySQL: false
BungeeCord: false

We have set up 3 channels; global, donor, and admin. We have removed any chat delay for all of these but it's only working in donor and admin. The global chat is delayed by a few seconds.

https://ufile.io/dv6uxhos - Debug

**Comment by ElBananaa** [comment]
> Minecraft Version: 1.16.4
> MySQL: false
> BungeeCord: false
> 
> We have set up 3 channels; global, donor, and admin. We have removed any chat delay for all of these but it's only working in donor and admin. The global chat is delayed by a few seconds.
> 
> https://ufile.io/dv6uxhos - Debug

Hey!
The debug file you provided shows us that CHC-Red and the dependencies are outdated.
Please update you plugins, your dependencies (At least update protocolLib) to the latest version and you server softwares (paper/spigot or whatever you use).

Once this is done, if the error persists, please upload us a new `/chc debug` file and your server latest.log.

**Comment by elkoyofficial** [comment]
Worked, ty.

**Comment by ElBananaa** [comment]
Alright, closing this issue, feel free to open a new issue if you need anything else! :)

**Comment by elkoyofficial** [comment]
It's back. Seems to work fine whenever we reload protocollib.

**Comment by ElBananaa** [comment]
> It's back. Seems to work fine whenever we reload protocollib.

Hey!
Please provide a new `/chc debug`
